### PR TITLE
Added cc and bcc rendering on message sent

### LINKED ISSUE #3473: missing cc: and bcc: in sent message rendering
if I send encrypted message:

```
to:person
cc:other 
```

then the sent message renders as just to:person without any cc. Only if I navigate away and open the message again, when the to/cc is rendered by google, will I see who the message was really sent. I would be good to render all.

### PR DESCRIPTION
**Pull request by martgil**
This PR will show cc and bcc email headers from a sent messages.

close #3473 // if this PR closes an issue

----------------------------------

**Tests** _(delete all except exactly one)_:
- Tests will be added later (issue #...)

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [x] addresses the issue it closes (if any)
- [x] code is readable and understandable
- [x] is accompanied with tests, or tests are not needed
- [x] is free of vulnerabilities
- [x] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

File: extension/chrome/elements/compose-modules/compose-send-btn-module.ts [hunk 1]
```diff
@@ -224,7 +225,9 @@ export class ComposeSendBtnModule extends ViewModule<ComposeView> {
     await this.view.draftModule.draftDelete();
     this.isSendMessageInProgress = false;
     if (this.view.isReplyBox) {
-      this.view.renderModule.renderReplySuccess(msg, msgSentRes.id);
+      const listOfCc = ComposeRenderModule.getEmailHeaderFromComposeMessageBox('cc');
```
Comment (limonte) [inline extension/chrome/elements/compose-modules/compose-send-btn-module.ts:228]: There's absolutely no need in this method. All recipients - `to`, `cc` and `ccc` - are available in `msg.recipients`

File: extension/chrome/elements/compose-modules/compose-render-module.ts [hunk 2]
```diff
@@ -122,6 +131,14 @@ export class ComposeRenderModule extends ViewModule<ComposeView> {
     this.view.S.cached('compose_table').css('display', 'none');
     this.view.S.cached('reply_msg_successful').find('div.replied_from').text(this.view.senderModule.getSender());
     this.view.S.cached('reply_msg_successful').find('div.replied_to span').text(msg.headers.To.replace(/,/g, ', '));
+    if (ccHeaders.length > 0) {
+      this.view.S.cached('reply_msg_successful').find('div.replied_cc span').text(ccHeaders.join(','));
```
Comment (limonte) [inline extension/chrome/elements/compose-modules/compose-render-module.ts:135]:
```diff
- this.view.S.cached('reply_msg_successful').find('div.replied_cc span').text(ccHeaders.join(','));
+ this.view.S.cached('reply_msg_successful').find('div.replied_cc span').text(ccHeaders.join(', '));
```

File: extension/chrome/elements/compose-modules/compose-render-module.ts [hunk 3]
```diff
@@ -122,6 +131,14 @@ export class ComposeRenderModule extends ViewModule<ComposeView> {
...
+      $('.replied_cc').show();
+    }
+    if (bccHeaders.length > 0) {
+      this.view.S.cached('reply_msg_successful').find('div.replied_bcc span').text(bccHeaders.join(','));
```
Comment (limonte) [inline extension/chrome/elements/compose-modules/compose-render-module.ts:139]: same as above

File: extension/css/cryptup.css [hunk 4]
```diff
@@ -2326,9 +2326,11 @@ div#reply_message_successful_container > table div.replied_from {
   margin-bottom: 4px;
 }
 
-div#reply_message_successful_container > table div.replied_to {
+div#reply_message_successful_container > table div.replied_to,
+table div.replied_cc,
```
Comment (limonte) [inline extension/css/cryptup.css:2330]:
```diff
- table div.replied_cc
+ div#reply_message_successful_container > table div.replied_cc
```

File: extension/css/cryptup.css [hunk 5]
```diff
@@ -2326,9 +2326,11 @@ div#reply_message_successful_container > table div.replied_from {
...
+table div.replied_bcc {
```
Comment (limonte) [inline extension/css/cryptup.css:2331]: same as above

**Review by limonte — CHANGES_REQUESTED** [review]

**Comment by limonte** [comment]
@martgil when you've added the requested changes, please reply with "done" or similar in the related conversations, thanks!

File: extension/chrome/elements/compose-modules/compose-send-btn-module.ts (hunk 1 above)
Comment (martgil) [reply extension/chrome/elements/compose-modules/compose-send-btn-module.ts]: Thank you @limonte, the `msg.recipients` contains all the necessary headers to be checked.

File: extension/css/cryptup.css (hunk 4 above)
Comment (martgil) [reply extension/css/cryptup.css]: I have done changing the incorrect CSS declaration here, thanks!

File: extension/chrome/elements/compose-modules/compose-render-module.ts (hunk 2 above)
Comment (martgil) [reply extension/chrome/elements/compose-modules/compose-render-module.ts]: I have done removing other unnecessary code and I use the `msg.recipients` instead, Thanks!

**Comment by martgil** [comment]
Yes sir, @limonte. Thanks!

File: extension/chrome/elements/compose-modules/compose-render-module.ts (hunk 2 above)
Comment (limonte) [reply extension/chrome/elements/compose-modules/compose-render-module.ts]: this was the request was about the missing space symbol:

CleanShot 2021-09-03 at 13 05 36@2x

Comment (limonte) [reply extension/chrome/elements/compose-modules/compose-render-module.ts]: @martgil ping

File: extension/chrome/elements/compose.htm [hunk 6]
```diff
@@ -167,6 +167,8 @@ <h1 id="header_title" data-test="header-title">New Secure Message</h1>
             <!-- to -->
             <div class="replied_from"></div>
             <div class="replied_to">to <span></span></div>
+            <div class="replied_cc display_none">cc <span></span></div>
+            <div class="replied_bcc display_none">bcc <span></span></div>
```
Comment (limonte) [inline extension/chrome/elements/compose.htm:171]: please add `:` in these places:

```diff
- <div class="replied_to">to <span></span></div>
+ <div class="replied_to">to: <span></span></div>

- <div class="replied_cc display_none">cc <span></span></div>
+ <div class="replied_cc display_none">cc: <span></span></div>

- <div class="replied_bcc display_none">bcc <span></span></div>
+ <div class="replied_bcc display_none">bcc: <span></span></div>
```

**Review by limonte — CHANGES_REQUESTED** [review]

**Review by limonte — APPROVED** [review]
LGTM

**Comment by martgil** [comment]
Thank you sir!

**Pull request merged**